### ISSUE TITLE
Redeploy to stage not working.

### ISSUE THREAD

**Issue by mccalluc** [hubmapconsortium/portal-ui#1342]
John pointed out that stage hadn't really redployed. Running redeploy.sh again, the output is:
```
$ ./redeploy.sh mccalluc stage
Pseudo-terminal will not be allocated because stdin is not a terminal.
whoami? mccalluc
whoami? root
whoami? centos
portal running? 0cfec587bf80 hubmap/portal-ui:latest "/entrypoint.sh /sta…" 15 hours ago Up 15 hours 80/tcp, 443/tcp portal-ui 17ac5fdd0240 hubmap/portal-ui:latest "/entrypoint.sh /sta…" 2 days ago Up 2 days 80/tcp, 443/tcp portal-ui-prod
stopping...
Stopping portal-ui ... done
Removing portal-ui ... done
Network gateway_hubmap is external, skipping
portal running? 17ac5fdd0240 hubmap/portal-ui:latest "/entrypoint.sh /sta…" 2 days ago Up 2 days 80/tcp, 443/tcp portal-ui-prod
removing old "latest"...
Error response from daemon: conflict: unable to remove repository reference "hubmap/portal-ui:latest" (must force) - container 17ac5fdd0240 is using its referenced image 1775a2b5d2ca
starting...
Creating portal-ui ...
portal running? 749c82003dd7 hubmap/portal-ui:latest "/entrypoint.sh /sta…" 1 second ago Up Less than a second 80/tcp, 443/tcp Creating portal-ui ... done
the input device is not a TTY
Last login: Mon Nov  2 22:09:57 UTC 2020
```
Note the `unable to remove repository reference "hubmap/portal-ui:latest" (must force)`. So I added an `-f` to the `rmi` and the output is now:
```
$ ./redeploy.sh mccalluc stage
Pseudo-terminal will not be allocated because stdin is not a terminal.
whoami? mccalluc
whoami? root
whoami? centos
portal running? 749c82003dd7 hubmap/portal-ui:latest "/entrypoint.sh /sta…" 4 minutes ago Up 4 minutes 80/tcp, 443/tcp portal-ui 17ac5fdd0240 hubmap/portal-ui:latest "/entrypoint.sh /sta…" 2 days ago Up 2 days 80/tcp, 443/tcp portal-ui-prod
stopping...
Stopping portal-ui ... done
Removing portal-ui ... done
Network gateway_hubmap is external, skipping
portal running? 17ac5fdd0240 hubmap/portal-ui:latest "/entrypoint.sh /sta…" 2 days ago Up 2 days 80/tcp, 443/tcp portal-ui-prod
removing old "latest"...
Untagged: hubmap/portal-ui:latest
Untagged: hubmap/portal-ui@sha256:d42cf22c9d80390c0769ee35f0ab98df36db9a157526f94af16192b431b1e938
starting...
Pulling portal-ui (hubmap/portal-ui:latest)...
latest: Pulling from hubmap/portal-ui
Digest: sha256:b1f173eeba7b20ce292e831c2e6a083eea0ae42b1ad7ed2ace7aff39b67f088a
Status: Downloaded newer image for hubmap/portal-ui:latest
Creating portal-ui ... done
portal running? ff028912757e hubmap/portal-ui:latest "/entrypoint.sh /sta…" Less than a second ago Up Less than a second 80/tcp, 443/tcp portal-ui 17ac5fdd0240 1775a2b5d2ca "/entrypoint.sh /sta…" 2 days ago Up 2 days 80/tcp, 443/tcp portal-ui-prod
the input device is not a TTY
Last login: Tue Nov  3 13:36:52 UTC 2020
```
But it is not finding the JS bundle, and https://portal.stage.hubmapconsortium.org/ is just blank.

The problem is just on stage. [dev](https://portal.dev.hubmapconsortium.org/CHANGELOG) and [test](https://portal.test.hubmapconsortium.org/CHANGELOG) redeployed successfully.

**Comment by yuanzhou** [comment]
This is due to the changes I made yesterday on STAGE by adding the portal-ui-prod which is using the same `hubmap/portal-ui:latest` image. A simple tweak would resolve this. In that redeploy.sh script, also stop and remove the portal-ui-prod container before deleting the image. Then start up both `portal-ui` and `portal-ui-prod` containers after downloading the updated image.
Basically I've created a new domain with ssl for another portal-ui instance based on the same image on STAGE. This `portal-ui-prod` is located at `/home/centos/hubmap/portal-ui-prod` under user `centos` on STAGE, to start up the container: docker-compose -f hubmap.yml up -d and to stop and delete the container: docker-compose -f hubmap.yml down . The same commands you used for the regular portal-ui .

And the new domain can be found here: https://github.com/hubmapconsortium/gateway/issues/57

This portal-ui-prod uses the same `hubmap/portal-ui:latest` image on STAGE only. But it points to the search-api, entity-api, and assets file services on PROD.
